starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +21,43 @@ var words = [
   'crackpot'
 ];
 
+
+function findLongestWord (words) {
+  var maxLengthWord = words[0].length;
+   for (var i = 0 ; i < words.length ; i++) {
+      if (words[i].length > maxLengthWord) {
```

Comment:
To stick to the expectation of the test, you should add an if that manage to return undifined if the array is empty